Update README.md

ericahagle opened the pull request:
# Description
- Updates README.md

# Notes
- Totally snagged some of @Sykogst 's awesome badges for this!
- Will still need to be updated
  - Installation instructions (to include server info)
  - Deploy link (when we have it)
  - Challenges/Wins
  - Abstract
  - App Preview

# Checklist
- [x] My PR has an appropriately descriptive and concise title.
- [x] My code follows the Turing Style Guides and best practices.
- [x] I ran the code locally and verified that there are no visible errors.
- [x] docs: My PR clearly describes which doc(s) I'm updating and why.

zkevkev reviewed (approved):
This looks fantastic to me. I don't see any areas for improvement currently and I'm super excited that we are going to have such professional READMEs.

Sykogst commented on README.md:
```diff
@@ -1,70 +1,196 @@
+<!-- TBD -->
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+# 📝 Context
```
I really like the context section and will nab this for the backend readme. I will be sure to link the frontend repo as you did for the backend.

ericahagle commented on README.md:
```diff
@@ -1,70 +1,196 @@
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+1. Navigate (`cd`) into your local directory containing the repository
+2. Run `npm install cypress` to install Cypress dependencies
```
I like this idea, and the reasoning for it. Will update!

tenthwalker reviewed (approved):
Unsure why it shows the README file as deleted and then recreated when I moved it to a new folder, but I created a new directory for documentation and updated the hyperlink to the backend repo to match the conventions of the rest of the links in the file.

ericahagle commented:
> Unsure why it shows the README file as deleted and then recreated when I moved it to a new folder, but I created a new directory for documentation and updated the hyperlink to the backend repo to match the conventions of the rest of the links in the file.

@tenthwalker Looks like stuff kinda got borked here, but I can fix it in the new branch I'm working on to set up the modular file structure. There are now two pr templates, and the link looks funky in the readme. 
<img width="890" alt="Screenshot 2024-02-01 at 9 08 19 PM" src="https://github.com/Tavern-Keeper-2308/tavern-keeper-fe/assets/133910120/0a900dea-00a6-49ee-b592-dc5c361327e4">
